src/porepy/models/constitutive_laws.py

```diff
@@ -9,7 +9,6 @@
 import scipy.sparse as sps
 
 import porepy as pp
```

Comment:
I'm down for keeping it with pp, because it makes it one step easier to copy and edit, for example when you want to construct a custom constitutive law: you don't need to copy the import line.